app/battle/battle_preparations.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+
+class Knight:
+    def __init__(self, knight: dict) -> None:
+        self.knight = knight
+
+    @staticmethod

```

Comment:
instead of implementing static methods better use self methods and its properties